src/app/routes/index.js

```diff
@@ -1,15 +1,18 @@
 import Article from '../containers/Article';
+import getInitialData from './getInitialData';
 
 const routes = [
   {
     path: '/:service/articles/:id',
     exact: true,
     component: Article,
+    getInitialData,
```

Comment:
Could you explain how the match props are passed to the callback of `getInitialData`?

Reply:
`uni` calls `react-router-config`'s `matchRoute`, which returns a `route` and `match` object. This is just passed to `getInitialData`. https://github.com/jtart/uni/blob/master/src/loadInitialData.js#L17

Reply:
@pjlee11 It's because in the `server/index` we pass the routes array to loadInitialData `const data = await loadInitialData(url, routes);` and here you can see it used in Uni's `loadInitialData` function: https://github.com/jtart/uni/blob/a3b7d204b9e623d6675396e45650ee3fc9a4751e/src/loadInitialData.js#L12-L21